src/components/map/RouteMap.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState, useEffect } from "react";
+import { RoutePoiButton } from "./RoutePoiButton";
+import "./RouteMap.scss";
+import "./RoutePoiButton.scss";
+
+interface PoiLocation {
+  x: number;
+  y: number;
+  sol: number;
+}
+
+interface RouteMapProps {
+  roverName: string;
+  mapImageUrl: string;
+  pois: PoiLocation[];
+}
+
+export const RouteMap: React.FunctionComponent<RouteMapProps> = ({
+  roverName,
+  mapImageUrl,
+  pois,
+}) => {
+  const [sol, setSol] = useState<number>();
+  const [photoArr, setPhotosArr] = useState<string[]>([]);
```

Comment:
Names should match, and also use the plural form of the noun instead of "thingArray"
```suggestion
  const [photoUrls, setPhotoUrls] = useState<string[]>([]);
```